libexec/basher-_clone

```diff
@@ -34,4 +42,6 @@ if [ -e "$BASHER_PACKAGES_PATH/$package" ]; then
   exit 0
 fi
 
-git clone --depth=1 --recursive "https://github.com/$package.git" "${BASHER_PACKAGES_PATH}/$package"
+[[ $proto == "ssh" ]] && repo=$login${login:+@}$site: || repo=$proto://$site/
+
+git clone --depth=1 --recursive "$repo$package".git "${BASHER_PACKAGES_PATH}/$package"
```

Comment:
I don't see that as a valid use case.  Presumably they are the same project if they have the same name, and you wouldn't clone to separate copies in that case, instead you would clone from one and then add the other as a new remote.